.github/workflows/googlecloudrun.yml

```diff
@@ -47,4 +47,4 @@ jobs:
         run: |
           gcloud config set project ${{ secrets.GCLOUD_PROJECT }}
           gcloud run deploy ${{ env.GCLOUD_APP_NAME }} --image gcr.io/${{ secrets.GCLOUD_PROJECT }}/${{ env.GCLOUD_APP_NAME }} --project ${{ secrets.GCLOUD_PROJECT }} --region us-west1 --platform managed --allow-unauthenticated 
-          gcloud run services update ${{ env.GCLOUD_APP_NAME }} --region us-west1 --add-cloudsql-instances ${{ secrets.CLOUD_SQL_CONNECTION_NAME }} --set-env-vars CLOUD_SQL_CONNECTION_NAME=${{ secrets.CLOUD_SQL_CONNECTION_NAME }},CLOUD_SQL_USERNAME=${{ secrets.CLOUD_SQL_USERNAME }},CLOUD_SQL_PASSWORD=${{ secrets.CLOUD_SQL_PASSWORD }},CLOUD_SQL_DATABASE_NAME=${{ secrets.CLOUD_SQL_DATABASE_NAME }} --platform managed
+          gcloud run services update ${{ env.GCLOUD_APP_NAME }} --platform managed --region us-west1 --add-cloudsql-instances ${{ secrets.CLOUD_SQL_CONNECTION_NAME }} --set-env-vars CLOUD_SQL_CONNECTION_NAME=${{ secrets.CLOUD_SQL_CONNECTION_NAME }},CLOUD_SQL_DATABASE_USERNAME=${{ secrets.CLOUD_SQL_DATABASE_USERNAME_FRONTEND }},CLOUD_SQL_DATABASE_PASSWORD=${{ secrets.CLOUD_SQL_DATABASE_PASSWORD_FRONTEND }},CLOUD_SQL_DATABASE_NAME=${{ secrets.CLOUD_SQL_DATABASE_NAME }} 
```

Comment:
@AaronMT since we'll eventually merge the frontend and backend database connector modules, I'm not exactly sure how we'll manage this but it will likely be a bit simpler than this